lib/ansible/modules/find.py

```diff
@@ -433,13 +440,18 @@ def main():
     now = time.time()
     msg = ''
     looked = 0
+
+    def handle_errors(e):
+        if params['warn']:
+            module.warn('find had a problem: ' + to_text(e))
```

Comment:
Maybe just have this raise the exception and then it will be caught and handled by the existing `except`, which will warn.

Reply:
we don't currently stop on these, this just adds reporting

Reply:
also made the function more generic